modules/hooks.nix

```
@@ -605,6 +605,17 @@ in
             ];
           entry = "${tools.shellcheck}/bin/shellcheck";
         };
+      bats =
+        {
+          name = "bats";
+          description = "Run bash unit tests.";
+          types = [ "shell" ];
+          types_or =
+            [
+              "bats"
```

```suggestion
              "bash"
```

Shouldn't it be `bash` instead of `bats` in `types_or`?

---

I'm not sure; I understand that this list the files that trigger this pre-commit hook.  Is this the case?

If that is the case, I wouldn't like to get all `.bash` files to get called with bats because it is very likely those are not bats test. By having only `bats` here, the changes to `.bats` file are the only ones that will get picked up by the hook.

---

On second thought, I tested bats running on regular bash files and it ignores them correctly, so, sure let me add that